### PR TITLE
🎉 New Source: Pingdom

### PR DESCRIPTION
**Pull request by KimPlv**
## What
Add a new source with [Pingdom API](https://docs.pingdom.com/api/)

## How
Extract PingDom streams through the API

## Why 
Interact  with the Pingdom system

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- Connector version is set to `0.1.0`
    - `Dockerfile` has version `0.1.0`
- Documentation updated
    - `docs/integrations/source/pingdom.md` including changelog with an entry for the initial version. 
    - `docs/integrations/README.md`

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-persistiq/source_persistiq/manifest.yaml [hunk 1]
```diff
@@ -1,73 +1,118 @@
-version: "0.29.0"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-persistiq/source_persistiq/manifest.yaml:1]: Any reason for this change?

File: airbyte-integrations/connectors/source-pingdom/source_pingdom/schemas/checks.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-pingdom/source_pingdom/schemas/checks.json:3]: Missing additionalProperties = true. @KimPlv can you run `airbyte-ci format fix all`?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
The implementation looks good. We need to have a sandbox account to run integration tests and validate the connector this can take some time, if you have one to share please contact me on Slack.

File: airbyte-integrations/connectors/source-pingdom/source_pingdom/schemas/checks.json (hunk 2 above)
Comment (KimPlv) [reply airbyte-integrations/connectors/source-pingdom/source_pingdom/schemas/checks.json:3]: done :white_check_mark:

**Comment by marcosmarxm** [comment]
Hi @KimPlv, apologies for the delay in updating and reviewing your contribution. Regrettably, we had to decline it for now because the team was unable to set up a sandbox and conduct integration tests. We understand this can be disappointing and we want to assure you that we recognize this issue. We dedicated a significant amount of effort in the previous quarter, and will continue to do so in the next to improve the process for contributions like new connectors. The team will include a new process for contributing and publishing sources in future doesn't depend on Airbyte needs to have a proper sandbox to accept contributions and ensure all tests are valid. Please get in touch if you want to learn more about it and resubmit the amazing source :) Thanks a lot for all effort you put into this contribution and feel free to contact me on Slack

**Pull request closed without merging**